landing-pages/site/content/en/ecosystem/_index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Ecosystem"
+linkTitle: "Ecosystem"
+menu:
+  main:
+    weight: 30
+---
+
+&nbsp;
+&nbsp;
+
+# Ecosystem
+
+Those resources and services are not maintained, nor endorsed by the Apache Airflow Community and Apache Airflow project (maintained by the Committers and the Airflow PMC). Use them at your sole discretion. The community does not verify the licences nor validity of those tools, so it's your responsibility to verify them.
+
+If you would you like to be included on this page, please reach out to the [Apache Airflow dev or user mailing list](https://airflow.apache.org/community/) and let us know or simply open a Pull Request to that page.
+
+&nbsp;
+
+## Learning resources
+
+[Apache Airflow YouTube Channel](https://www.youtube.com/channel/UCSXwxpWZQ7XZ1WL3wqevChA) - Official YouTube Channel
+
+[Airflow Summit](https://airflowsummit.org/) - Online conference for Apache Airflow developers
+
+[Awesome Apache Airflow](https://github.com/jghoman/awesome-apache-airflow) - Curated list of resources about Apache Airflow
+
+[The Complete Hands-On Introduction to Apache Airflow](https://www.udemy.com/course/the-complete-hands-on-course-to-master-apache-airflow) by Marc Lamberti on Udemy
+
+[Apache Airflow: Complete Hands-On Beginner to Advanced Class](https://www.udemy.com/course/apache-airflow-course) by Alexandra Abbas on Udemy
+
+&nbsp;
+
+## Airflow-As-A-Service
+
+[Astronomer](https://www.astronomer.io/) - Managed Apache Airflow service on Astronomer Cloud
+
+[Google Cloud Composer](https://cloud.google.com/composer) - Managed Apache Airflow service on Google Cloud Platform
+
```

Comment:
Can we also add `[Qubole](https://qubole.com) - Managed Apache Airflow Service on all major public clouds`

Reply:
BTW. They have old logo of Airflow @ Qubole :)

Reply:
I added Qubole as well :)